sycl/test/usm/findplatforms.hpp

```diff
@@ -21,14 +21,15 @@ bool findPlatformAndDevice(cl_device_type deviceType,
     cl_uint numDevices = 0;
     errorCode =
       clGetDeviceIDs(platform, deviceType, 0, nullptr, &numDevices);
-    if (errorCode != CL_SUCCESS) return false;
-
-    std::vector<cl_device_id> devices(numDevices);
-    errorCode = clGetDeviceIDs(platform, deviceType, numDevices,
-                               devices.data(), nullptr);
-    if (errorCode != CL_SUCCESS) return false;
+    if (!(errorCode == CL_SUCCESS || errorCode == CL_DEVICE_NOT_FOUND))
```

Comment:
Could you, please, justify this change?

Reply:
Sure.  clGetDeviceIDs is stupid.  I'll add a comment.